packages/dds/merge-tree/src/test/mergeTree.insertingWalk.spec.ts

```diff
@@ -157,7 +157,7 @@ const treeFactories: ITestTreeFactory[] = [
                 initialText.length,
                 UniversalSequenceNumber,
                 localClientId,
-                UnassignedSequenceNumber,
+                UniversalSequenceNumber,
```

Comment:
test was wrong. we should use UnassignedSequenceNumber before start collaboration is called